etrago/appl.py

```diff
@@ -217,186 +221,190 @@ def run_etrago(args, json_path):
+        The other options affect the kmeans algorithm and should only be
+        changed carefully, documentation and possible settings are described
+        in sklearn-package (sklearn/cluster/k_means_.py).
+        This function doesn't work together with ``'line_grouping = True'``.
+
+    sector_coupled_clustering : nested dict
+        {'active': True, 'carrier_data': {
+         'H2_ind_load': {'base': ['H2_grid']},
+         'central_heat': {'base': ['CH4']},
+         'rural_heat': {'base': ['CH4']}}
+        }
+        State if you want to apply clustering of sector coupled carriers, such
+        as central_heat or rural_heat. The approach builds on already clustered
+        buses (e.g. CH4 and AC) and builds clusters around the topology of the
+        buses with carrier ``'base'`` for all buses of a specific carrier, e.g.
+        ``'H2_ind_load'``.
+
+    network_clustering_ehv : bool
+        False,
+        Choose if you want to cluster the full HV/EHV dataset down to only the
+        EHV buses. In that case, all HV buses are assigned to their closest EHV
+        sub-station, taking into account the shortest distance on power lines.
+
+    snapshot_clustering : dict
+        {'active': False, 'method':'typical_periods', 'how': 'daily',
+         'storage_constraints': '', 'n_clusters': 5, 'n_segments': 5},
+        State if you want to apply a temporal clustering and run the optimization
+        only on a subset of snapshot periods.
+        You can choose between a method clustering to typical periods, e.g. days
+        or a method clustering to segments of adjacent hours.
+        With ``'how'``, ``'storage_constraints'`` and ``'n_clusters'`` you choose
+        the length of the periods, constraints considering the storages and the number
+        of clusters for the usage of the method typical_periods.
+        With ``'n_segments'`` you choose the number of segments for the usage of
+        the method segmentation.
+
+    branch_capacity_factor : dict
+        {'HV': 0.5, 'eHV' : 0.7},
+        Add a factor here if you want to globally change line capacities
+        (e.g. to "consider" an (n-1) criterion or for debugging purposes).
+
+    load_shedding : bool
+        False,
+        State here if you want to make use of the load shedding function which
+        is helpful when debugging: a very expensive generator is set to each
+        bus and meets the demand when regular
+        generators cannot do so.
+
+    foreign_lines : dict
+        {'carrier':'AC', 'capacity': 'osmTGmod}'
+        Choose transmission technology and capacity of foreign lines:
+            'carrier': 'AC' or 'DC'
+            'capacity': 'osmTGmod', 'ntc_acer' or 'thermal_acer'
+
+    comments : str
+        None
+
+    Returns
+    -------
+    network : `pandas.DataFrame<dataframe>`
+        eTraGo result network based on `PyPSA network
+        <https://www.pypsa.org/doc/components.html#network>`_
+
```

Comment:
It looks like many lines were changes here, but I can not really see the difference. Do you have any ideas?